examples/grpc-reactor/src/test/java/example/armeria/grpc/reactor/HelloServiceTest.java

```diff
@@ -128,4 +131,18 @@ void bidirectionalHello() {
             assertThat(messages.get(i)).isEqualTo("Hello, " + names[i] + '!');
         }
     }
+
+    @Test
+    void exceptionMapping() {
```

Comment:
One of my TODOs is to clean up some test cases added for testing Armeria functions.
(I've already added tests to examples 🙈)
Would you mind moving this case to `it/grpc/kotlin`?

Reply:
Only this test case?

Reply:
By the way, this is `reactor-grpc` example. 😅
 
I think we can regard the test cases in `examples/grpc` as a standard test suite.

The following test cases and related gRPC service methods should be moved to it-grpc.
- `examples/gprc-reactor`
  - getLotsOfRepliesWithoutScheduler()
- `examples/grpc-kotlin`
  - parallelReplyFromServerSideBlockingCall()
  - parallelBlockingLotsOfReplies()
  - parallelShortBlockingLotsOfReplies()

Reply:
Done